applications/GeoMechanicsApplication/custom_utilities/constitutive_law_utilities.cpp

```diff
@@ -99,4 +99,38 @@ Matrix ConstitutiveLawUtilities::MakeInterfaceConstitutiveMatrix(double      Nor
     return result;
 }
 
+void ConstitutiveLawUtilities::CheckStrainSize(const Properties&               rProperties,
+                                               const std::vector<std::size_t>& rExpectedStrainSizes,
+                                               std::size_t                     Dim,
+                                               std::size_t                     ElementId)
+{
+    const std::size_t strain_size = rProperties[CONSTITUTIVE_LAW]->GetStrainSize();
+    const auto        size_exists =
+        std::any_of(rExpectedStrainSizes.begin(), rExpectedStrainSizes.end(),
+                    [strain_size](std::size_t expected_size) { return strain_size == expected_size; });
+    if (!size_exists) {
```

Comment:
As mentioned before, after only passing one number (e.g. as ExpectedSize), this can be simplified to 
```suggestion
    if (strain_size != ExpectedSize) {
```